### ISSUE TITLE
Publishing apk in beta track causes the step to fail

### ISSUE THREAD

**Issue by kmiralles** [bitrise-steplib/steps-google-play-deploy#12]
When switched back to alpha track works again without problems...
```
+------------------------------------------------------------------------------+
| (1) Publish APK to Google Play                                   |
+------------------------------------------------------------------------------+
| id: google-play-deploy                                                       |
| version: 1.0.0                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: go                                                                  |
| time: 2016-11-18T20:24:37Z                                                   |
+------------------------------------------------------------------------------+
|                                                                              |
INFO[20:24:37] Start installing (golang) with apt-get       
INFO[20:24:37]  * [OK] Step dependency (go) installed, available. 

Configs:
- JSONKeyPath: file:///bi***son
- PackageName: com.**********.***
- ApkPath: /bitrise/deploy/app-prod-release.apk
- Track: beta
- UserFraction: 0.5
Deprecated Configs:
- ServiceAccountEmail: 
- P12KeyPath: 

Authenticateing
Authenticated client created

Create new edit
 editID: 0030827******606545

Upload apks
 uploaded apk version: 5

Update track
 updated track: beta
 assigned apk versions: [5]
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x404c40]

goroutine 1 [running]:
panic(0x675740, 0xc42000c0c0)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
main.main()
	/tmp/bitrise-go-toolkit725175247/src/github.com/bitrise-steplib/steps-google-play-deploy/main.go:407 +0x11e0
|                                                                              |
+---+---------------------------------------------------------------+----------+
| x | Publish APK to Google Play Alpha Track (exit code: 2)         | 18 sec   |
+---+---------------------------------------------------------------+----------+
| Issue tracker: https://github.com/bitrise-io/steps-google-play-deploy/issues |
| Source: https://github.com/bitrise-io/steps-google-play-deploy               |
+---+---------------------------------------------------------------+----------+
```

Any idea what could be going wrong here?

Thanks in advance,

Federico

**Comment by viktorbenei** [comment]
Thanks for reporting @kmiralles !

@godrei please schedule a fix for this!

**Comment by godrei** [comment]
Hi @kmiralles ,
we released a new version [1.0.1](https://github.com/bitrise-steplib/steps-google-play-deploy/releases/tag/1.0.1), which fixes this issue, please update your step version and run a build again.

Sorry for the inconvenience!
